Yolink: add network information (#33931)
home-assistant/home-assistant.github.io: source/_integrations/yolink.markdown

If you want to use separate credentials, please contact <service@yosmart.com> to obtain a `client_id` and `client_secret`. Then you can add your credentials via application credentials. Settings > Devices & Services > click the menu (three dots at the top right of the screen) and then **Application Credentials**. Enter your credentials in the pop-up window.
{% enddetails %}

> {% warning %}

Reviewer: Thank you for the comments, @matrixd2 :+1: 
I think this could be placed in a section at the bottom of the page, called ## Community notes.
I would make a suggestion, but it is too far outside for me to let me add it.
Another note: If you fix the space in "Home Assistant", the linter will pass.

- YS8004-UC (Weatherproof Temperature Sensor)
- YS8005-UC (Weatherproof Temperature & Humidity Sensor)
- YS8006-UC (X3 Temperature & Humidity Sensor)
- YS8007-UC (Thermometer Hygrometer)
- YS8014-UC (X3 Outdoor Temperature Sensor)
- YS8015-UC (X3 Outdoor Temperature & Humidity Sensor)
- YS5006-UC (FlowSmart Control)
- YS5007-UC (FlowSmart Meter)
- YS5008-UC (FlowSmart All-in-One)

## Services

### `Play on SpeakerHub`

With this service, you can convert text to speech for playback on SpeakerHub.

Service data attribute | Optional | Description
-|-|-
`target_device` | no| SpeakerHub device ID for audio playback.
`message` | no| Text for speech conversion.
`tone` | no| Tone before playing audio.
`volume` | no| Speaker volume during playback.
`repeat` | no| The number of times the text will be repeated.

## Community notes

> 1. Integration requires an MQTT connection to be established via port 8003. If you are using a firewall, please allow communication via port 8003 in the firewall settings.  

Reviewer: ```suggestion
1. This integration requires an MQTT connection to be established via port 8003. If you are using a firewall, please allow communication via port 8003 in the firewall settings.  
```

> 2. If you use a network proxy, such as a VPN, the integration may not be able to update the device status, please turn off the VPN.

Reviewer: ```suggestion
2. If you use a network proxy, such as a VPN, the integration may not be able to update the device status. Turn off the VPN.
```

> 3. Please do not use UAC as credentials for HomeAssistant YoLink integration.

Reviewer: ```suggestion
3. Please do not use UAC as credentials for the Home Assistant YoLink integration.
```
